web/projectsEvaluation.php

```diff
@@ -394,13 +394,41 @@ function loadProjects() {
             text: 'Extended view',
             handler: showExtendedView,
         }],
+        initComponent: function() {
+            this.tbar = this.buildTopToolbar();
+            Ext.ux.ExportableGridPanel.superclass.initComponent.apply(this, arguments);
+        },
+        buildTopToolbar: function () {
+            return [{
+                text: 'Details',
+                id: this.id + 'DetailsBtn',
+                ref: '../detailsBtn',
+                disabled: true,
+                iconCls: this.iconCls + '-go',
+                handler: this.onDetails,
+                scope: this
+            }]
+        },
+        /**
+         * onDetails
+         */
+        onDetails: function() {
+            if (this.getSelectionModel().getCount() > 0) {
+                var selected = this.getSelectionModel().getSelected();
+                window.location = 'viewProjectDetails.php?pid=' + selected.id;
+            }
+        }
```

Comment:
And please, open the project details in a new window. That was an explicit user request in #133.